pkgs/development/libraries/libdbi-drivers/default.nix

```diff
@@ -16,6 +16,10 @@ stdenv.mkDerivation rec {
 
   buildInputs = [ libdbi sqlite postgresql ] ++ lib.optional (libmysqlclient != null) libmysqlclient;
 
+  patches = [
```

Comment:
I suppose this can't be `fetchpatch` ed.
Can you include a comment with the origin URL?

Reply:
Done. Force pushed 88801dab5eff529e79f066a7bb230842b876ccb1 .